packages/xrpl/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "pretty": true,
-    "target": "es6",
+    "target": "es2021",
```

Comment:
Do we want to update to such a new standard?

Reply:
es2017 is the minimum recommendation. I can use that instead? (Not sure what's best)

Reply:
Yeah just wanted to warn.  It could be fine.  I just have never targeted 2021.

Reply:
Will this affect people who are using a lower standard? We want to make sure we're backwards compatible

Reply:
This in fact could be a non-backwards compatible change, so I'm undoing it: https://stackoverflow.com/questions/73387445/what-happens-if-tsconfig-targets-and-libs-are-mixed-among-dependencies-for-t

Seems the error I was getting which prompted the upgrade was a trickle down error from my other configuration changes, since it seems to have disappeared since I moved the `@types/node` import to the top level and made it match the earliest version of node we support.